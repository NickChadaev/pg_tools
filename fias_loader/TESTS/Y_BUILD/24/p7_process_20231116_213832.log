2023-11-16 21:38:32 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_24 postgres /home/n.chadaev@abrr.local/Y_BUILD/24/stage_gar_c_24.csv 2023-11-13
--------------------------------------------------------
[21:38:32] ... Красноярский край, id_region: 25, Execute: /home/n.chadaev@abrr.local/Y_BUILD/24/stage_gar_c_24.csv, process_id:25345
[21:38:32] ... Start process
[21:38:32] ...  -- Version
[21:38:32] ...  -- Очистка от данных
[21:38:32] ...  -- Убираем индексы
[21:38:32] ...  -- Set UNLOGGED
[21:38:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_APARTMENT_TYPES_20231113_78e8cff6-619c-4b2e-806f-6c3c18c41d8c.XML
[21:38:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDR_OBJ_TYPES_20231113_15f2ec37-c925-4252-b49a-5473d975a16f.XML
[21:38:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ROOM_TYPES_20231113_32952955-cd74-40c1-bc84-06d94fe9559d.XML
[21:38:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OPERATION_TYPES_20231113_5151274c-91f4-4286-aaff-fc37112b660a.XML
[21:38:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_PARAM_TYPES_20231113_fe767969-20f3-4044-9ef6-67b4cb2881ce.XML
[21:38:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_HOUSE_TYPES_20231113_507d1000-4460-4d2f-a5b1-2ab5b9cd8a26.XML
[21:38:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDHOUSE_TYPES_20231113_d99b31fe-7b05-400a-ac92-23919e58da07.XML
[21:38:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OBJECT_LEVELS_20231113_b5fa7b3d-ea26-40c9-8b3a-9b986261ea66.XML
[21:38:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_TYPES_20231113_b8068ed2-dd18-4365-abb7-3dbd98698c35.XML
[21:38:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_KINDS_20231113_2f0f38c6-d598-4fd7-b325-2aa6aca29ed2.XML
[21:38:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/24/AS_REESTR_OBJECTS_20231113_cf66a2c6-848e-431c-b6b3-5c20c9f62609.XML
[22:09:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/24/AS_ADDR_OBJ_20231113_31c91690-ea18-40fb-a5df-51dfe3a1937a.XML
[22:10:16] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/24/AS_HOUSES_20231113_6dfa4e28-6ddb-4361-a4c3-2fbab4299c05.XML
[22:27:01] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/24/AS_STEADS_20231113_df512567-9287-4f90-8290-3483bfd168d1.XML
[22:34:40] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/24/AS_APARTMENTS_20231113_bdaeea3f-b978-4a7f-b36e-c4d8269c415e.XML
[22:50:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/24/AS_ROOMS_20231113_ba465923-704e-4179-bf70-514520c28ad1.XML
[22:50:31] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/24/AS_CARPLACES_20231113_bf768441-ec13-4c07-ba49-16d82ab1c1ca.XML
[22:50:39] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/24/AS_ADM_HIERARCHY_20231113_2780add7-6681-4798-ba52-d9dfdc0266c5.XML
[23:26:34] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/24/AS_MUN_HIERARCHY_20231113_835c42f5-44ed-4e12-a7ee-473af76a6cc4.XML
[00:01:05] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/24/AS_CHANGE_HISTORY_20231113_fbf706b0-20ad-487c-8ab7-3ba847f8f7fe.XML
[01:29:52] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/24/AS_ADDR_OBJ_DIVISION_20231113_e33dd997-2a74-48b5-8341-148a2bf1399b.XML
[01:29:52] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/24/AS_ADDR_OBJ_PARAMS_20231113_6690b9dd-6682-4809-8d45-cbf71e48fcba.XML
[01:36:03] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/24/AS_STEADS_PARAMS_20231113_6e0406ab-fc68-4f7e-9174-9955fa2ecee4.XML
[02:56:03] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/24/AS_HOUSES_PARAMS_20231113_087a1756-29b1-4f08-a391-daa1e4d1fb3b.XML
[04:45:24] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/24/AS_APARTMENTS_PARAMS_20231113_ca0e83e9-8dc7-49a1-bb3e-fe8220e70641.XML
[05:20:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/24/AS_ROOMS_PARAMS_20231113_ab7810e5-ca3b-4c0f-bcae-7fc56313564c.XML
[05:20:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/24/AS_CARPLACES_PARAMS_20231113_7fe487d9-e253-4c50-8309-ff49d6ef13cd.XML
[05:20:34] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/24/AS_NORMATIVE_DOCS_20231113_1fcdbdc1-8feb-4052-a2a5-7edafcb2d156.XML
[05:24:20] ...  -- Data finish
[05:24:20] ...  -- Set LOGGED
[05:25:26] ... Stop process
--------------------------------------------------------
