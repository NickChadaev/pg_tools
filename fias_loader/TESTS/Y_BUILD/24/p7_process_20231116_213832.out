 save_gar_version 
------------------
               81
(1 row)

CALL
CALL
CALL
 set_gar_dt_create 
-------------------
                81
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 2586310
gar_fias.as_addr_obj: 48244
gar_fias.as_houses: 1246818
gar_fias.as_steads: 604233
gar_fias.as_apartments: 1254462
gar_fias.as_rooms: 5677
gar_fias.as_carplaces: 9996
gar_fias.as_adm_hierarchy: 2784577
gar_fias.as_mun_hierarchy: 2756890
gar_fias.as_change_history: 7304036
gar_fias.as_addr_obj_division: 74
gar_fias.as_addr_obj_params: 508979
gar_fias.as_steads_params: 6610254
gar_fias.as_houses_params: 9045683
gar_fias.as_apartments_params: 2796673
gar_fias.as_rooms_params: 9755
gar_fias.as_carplaces_params: 868
gar_fias.as_normative_docs: 305237
